Desktop portfolio version

Estherstarr opened the pull request:
This is a Desktop version of my Portfolio. 
- Use media queries to change the size and appearance of the site when viewed with a desktop browser.

Estherstarr committed:
Add classes to support media query for desktop version

Estherstarr committed:
Add media query for desktop version and style

ShoiraTa commented on style.css:
```diff
@@ -594,3 +596,122 @@ textarea {
+  .body-m1 {
+    width: 80%;
+    margin: 0 auto;
+  }
```
- [ ] There is some gray space on both sides of the page this is happening because you have a width of 80%. Please make sure that the page takes full width in all sections, as it is given in Figma. 

Your design  | Figma design
------------- | -------------
<img width="741" alt="Screenshot 2022-02-25 at 18 52 08" src="https://user-images.githubusercontent.com/77038610/155736043-29c88272-bda4-4b0c-8d57-3b8151bc32c6.png"> | <img width="975" alt="Screenshot 2022-02-25 at 18 51 29" src="https://user-images.githubusercontent.com/77038610/155735925-1987b30f-4c30-4d28-863a-2bcf3a44c174.png">

ShoiraTa commented on index.html:
```diff
@@ -1,4 +1,4 @@
- <!DOCTYPE html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
```
- [ ] Please make sure to add the background elements as provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="741" alt="Screenshot 2022-02-25 at 18 52 08" src="https://user-images.githubusercontent.com/77038610/155736043-29c88272-bda4-4b0c-8d57-3b8151bc32c6.png"> | <img width="975" alt="Screenshot 2022-02-25 at 18 51 29" src="https://user-images.githubusercontent.com/77038610/155735925-1987b30f-4c30-4d28-863a-2bcf3a44c174.png">

sabetech committed:
Modify styles to include css transitions
-Change Project Button properties when mouse hovers on it.
-Change Image size when mouse hovers on it.

ShoiraTa commented on index.html:
```diff
@@ -7,270 +7,451 @@
-        </ul>
-        <div class="btn-container">
-          <button type="button" class="view-project-btn">See Project</button>
+    <div class="body-m1">
```
- [ ] The page is divided into two parts, half of it contains the `project's picture` and the other half contains the `project's description,` these should switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).
But in your case all projects have the same layout, please make sure that projects layout change places like it is provided in Figma: 


Your design  | Figma design
------------- | -------------
<img width="783" alt="Screenshot 2022-02-25 at 18 57 40" src="https://user-images.githubusercontent.com/77038610/155736914-cae1fb8f-d119-40c0-afda-0dc4d106a871.png">  | <img width="744" alt="Screenshot 2022-02-25 at 18 58 03" src="https://user-images.githubusercontent.com/77038610/155736981-8ea4da20-ab7c-4619-9963-e594441bf561.png">

ShoiraTa reviewed (changes requested):
Hi @sabetech ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.giphy.com/media/dEPGknRiEniec/giphy.gif" width="500" height="250" />

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my username. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sabetech committed:
Revert "Modify styles to include css transitions"
Because I committed in a wrong branch

This reverts commit caf2e0ac937a2015e901d7b0507f6d4a67e17740.

Estherstarr committed:
Adjust project layout in the index.html

Estherstarr committed:
Style the project layout

Estherstarr committed:
Add background image

Estherstarr committed:
Update git and github files

Estherstarr commented:
Thank you @ShoiraTa 
Required changes made:
- Remove gray space on both sides of the page.
- Add a background image to the header section.
-  Style the project layout.

medaminedev66 reviewed (approved):
# Hi @Estherstarr , 

### Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- The MIT license link is broken, kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.


**You may merge the changes**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Smahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
